### PR TITLE
added Operating systems and two courses in free-courses-hi.md

### PR DESCRIPTION
**Pull request by swastikgithub09**
## What does this PR do?
Add resource(s) 

### Description
added operating system section and inserted two courses in it by different creators
### Why is this valuable (or not)?
Developers need to have good knowledge of operating system. Also, there was no course related to operating system
### How do we know it's really free?
on youtube for free
### For book lists, is it a book? For course lists, is it a course? etc.
course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -196,6 +197,12 @@
 * [Machine Learning Tutorials For Beginners Using Python in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG) - CodeWithHarry (YouTube)
 
 
+### Operating Systems
+
+* [Operating System(Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiGz9donHRrE9I3Mwn6XdP8p) - Gate Smashers (YouTube)
+* [Operating Systems for Placements](https://www.youtube.com/playlist?list=PLDzeHZWIZsTr3nwuTegHLa2qlI81QweYG) - CodeHelp-byBabbar (YouTube)
```
Comment (eshellman) [inline courses/free-courses-hi.md:203]: title is 'Operating Systems Placements Course 2022'

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.

File: courses/free-courses-hi.md (hunk 1 above)
Comment (swastikgithub09) [reply courses/free-courses-hi.md]: thanks sir, but the given title is 'Operating Systems Placements Course 2022' so i updated old one with it.

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -196,6 +197,12 @@
 * [Machine Learning Tutorials For Beginners Using Python in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG) - CodeWithHarry (YouTube)
 
 
+### Operating Systems
+
+* [Operating System (Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiGz9donHRrE9I3Mwn6XdP8p) - Gate Smashers, `tch.:` Varun Singla
```
Comment (eshellman) [inline courses/free-courses-hi.md:202]:
```suggestion
* [Operating System (Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiGz9donHRrE9I3Mwn6XdP8p) - Gate Smashers
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -196,6 +197,12 @@
...
+* [Operating Systems for Placements 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTr3nwuTegHLa2qlI81QweYG) - CodeHelp, `tch.:` Love Babbar, `tch.:` Lakshay Kumar
```
Comment (eshellman) [inline courses/free-courses-hi.md:203]:
```suggestion
* [Operating Systems for Placements 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTr3nwuTegHLa2qlI81QweYG) - CodeHelp - by Babbar
```

**Review by eshellman — CHANGES_REQUESTED** [review]
use the byline given on the resource. Sorry for the back and forth.

**Pull request merged**